style.css

```diff
@@ -93,10 +79,36 @@ img:hover{
   float: left;
 }
 
-#closing{
-  color:rgb(231,84,128);
+#closing {
+  color: rgb(231, 84, 128);
 }
 
-#candidates{
+#candidates {
   border: 10px solid pink;
-}
\ No newline at end of file
+}
+
+@media screen and (max-width: 600px) {
+  #tribute-link {
+    position: initial;
+    left: initial;
+    top: initial;
+  }

```

Comment:
Reset the positioning for this. As with the float, the relative positioning you've applied shouldn't be necessary.